refined/shared/src/main/scala/monocle/refined/strings.scala

```diff
@@ -8,16 +8,17 @@ object strings extends StringsInstances
 
 trait StringsInstances {
 
-  def startsWith(prefix: String): Prism[String, StartsWithString[prefix.type]] = {
-    Prism.partial[String, Refined[String, StartsWith[prefix.type]]] {
-      case string if string.startsWith(prefix) => Refined.unsafeApply(string)
-    }{_.value}
-  }
+  def startsWith(prefix: String)(implicit v: Validate[String, StartsWith[prefix.type]]): Prism[String, StartsWithString[prefix.type]] =
+    refinedPrism[String, StartsWith[prefix.type]](prefix)
```

Comment:
I think you can omit the type parameters to `refinedPrism` here, type inference will figure them out.